sites.md

```diff
@@ -0,0 +1,247 @@
+
+Optional params:
+- `team_id`: if present filter entries by team_id.
+- `updated_after`: if present only return entries updated after given date. Valid values are dates in ISO8601 format.
+- `orderby`: if present allow specify the response order by providing one order clause
+  made of `<field_name> <direction>`. Where direction is `asc` or `desc` and `field_name` is one of the list below:
+    - `updated_at`.
+
+examples:
+  - `?orderby=updated_at+asc`
+  - `?orderby=updated_at+desc`
+
+```json
+GET /api/v4/sites?team_id=26&updated_after=2022-05-24T19:30:30Z&orderby=updated_after+asc&fields=-id,-team_id,-boundaries,-job_number,-address,-site_owner_id,-created_at
+```
+
+```json
+200 OK
+
+[
+  {
+    "uuid": "1b14cac0-bd7f-11ed-9b4e-acde48001122",
+    "name": "Doctor Jack-Jack",
+    "location": {
+      "type": "Point",
+      "coordinates": [
+        174.77604159462516,
+        -41.286319623500816
+      ]
+    },
+    "updated_at": "2023-03-08 20:01:53 +1300"
+  },
+  {
+    "uuid": "f36290bc-bd82-11ed-9b4e-acde48001122",
+    "name": "Captain Ant Claw",
+    "location": {
+      "type": "Point",
+      "coordinates": [
+        77.61010151368907,
+        75.81002131462157
+      ]
+    },
+    "updated_at": "2023-03-07 10:11:12 +1300"
+  }
+]
+```
+
+
+### Create a site
+Creates one site.
+
+##### Input fields for create:
+  - **uuid**
+  - **team_id**
+  - **name**
+  - **site_owner_id**
+  - job_number
+  - address
+  - location
+  - boundaries
+
+```json
+POST /api/v4/sites
+
+{
+  "site": {
+    "uuid": "1b14cac0-bd7f-11ed-9b4e-acde48001122",
+    "team_id": 26,
+    "name": "Doctor Jack-Jack",
+    "site_owner_id": 35,
+    "job_number": "5af30ff1ce",
+    "address": "Level 3/354 Lambton Quay, Wellington Central, Wellington 6011",
+    "location": "Point(174.77604159462516 -41.286319623500816)",
+    "boundaries": "POLYGON ((174.7760662374219 -41.28628888757578, 174.77599163848276 -41.2863095461495, 174.77600806701338 -41.286352248914284, 174.77608819800787 -41.2863304566517, 174.7760662374219 -41.28628888757578))"
+  }
+}
+```
+
+```json
+201 Created
+
+{
+  "id": 14,
+  "uuid": "1b14cac0-bd7f-11ed-9b4e-acde48001122",
+  "name": "Doctor Jack-Jack",
+  "site_owner_id": 35,
+  "team_id": 26,
+  "job_number": "5af30ff1ce",
+  "address": "Level 3/354 Lambton Quay, Wellington Central, Wellington 6011",
+  "location": {
+    "type": "Point",
+    "coordinates": [
+      174.77604159462516,
+      -41.286319623500816
+    ]
+  },
+  "boundaries": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          174.7760662374219,
+          -41.28628888757578
+        ],
+        [
+          174.77599163848276,
+          -41.2863095461495
+        ],
+        [
+          174.77600806701338,
+          -41.286352248914284
+        ],
+        [
+          174.77608819800787,
+          -41.2863304566517
+        ],
+        [
+          174.7760662374219,
+          -41.28628888757578
+        ]
+      ]
+    ]
+  },
```

Comment:
> Note to avoid confusion from the reader: conversation above was inconclusive about serialization(server output) It got to a definition for server inputs.

@aduuub , 
I am assuming for now that at least initially we will have lots of simple polygons drawn for boundaries. And not many with holes or even multiple polygons if any.
This piece of data may be downloaded and stored only for sites of interest (signed in, from same team, or some other rule lat will limit substantially their numbers)

Location seem that they will always be simple points and for the previous conversation we believe it will be fetched and stored a lot.

For this reason we decided to use the shortest valid input as location output which is the array with longitude an latitude. `[12.34, 56.78]`

Reply:
For boundaries field output from the server I am still to decide between WKT and GeoJSON.

GeoJSON: Large Payload but directly pluggable on Maps.
WKT: Shorter Payload but need extra parsing step per shape to be used on Maps